Test/comp/firstSteps/7_Dt_Algebraic.dfy.go.expect

```diff
@@ -0,0 +1,11 @@
+List.Nil
+List.Cons(5, List.Nil)
+(List.Nil, List.Cons(5, List.Nil))
+(List.Cons(5, List.Nil), List.Nil)
+()
+5 List.Nil
+List.Cons(0, List.Cons(1, List.Cons(2, List.Cons(3, List.Cons(4, List.Cons(5, List.Cons(6, List.Nil)))))))
+0 + 1 + 2 + 3 + 4 + 5 + 6 == 21 (once more, that's 21)
+{Berry.Smultron, Berry.Jordgubb, Berry.Hallon}
```

Comment:
I see your point here :-)
I think we can also leave the printing of sets unspecified (as in theory, there is no order in set), but have a function setToSeq that takes an order on the elements and transforms it into a sequence. No needed now.

Reply:
Yup, it's definitely a solvable problem in stdlib code (and we should provide good solutions). For testing we may want to transition to using `expect` more and diffing print output less instead.